app/mm-redux/selectors/entities/apps.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {GlobalState} from '@mm-redux/types/store';
+import {AppBinding} from '@mm-redux/types/apps';
+
+export function getAppsBindings(state: GlobalState, location?: string): AppBinding[] {
+    if (!state.entities.apps.bindings) {
+        return [];
+    }
+
+    if (location) {
+        const headerBindings = state.entities.apps.bindings.filter((b) => b.location === location);
+        return headerBindings.reduce((accum: AppBinding[], current: AppBinding) => accum.concat(current.bindings || []), []);
```

Comment:
Not really. At this moment (no cache on client side) these will be fetch just once on each channel navigation, and they are potentially different each time.

What they could use is better naming, since these are not headerBindings. Just bindings.